### ISSUE TITLE
Support switching the assembly syntax of the disassembly view

### ISSUE THREAD

**Issue by a1q123456** [microsoft/vscode-cpptools#8869]
Type: Feature Request

Currently, the disassembly view shows the AT&T assembly syntax. So, I'm wondering if we could add an option for switching the assembly syntax between Intel and AT&T?

Thank you.

**Comment by WardenGnaw** [comment]
In the `Debug Console` you can execute gdb-mi commands to switch the assembly syntax.

E.g.
`-gdb-set disassembly-flavor intel` or `-gdb-set disassembly-flavor att`

[Reference](https://visualgdb.com/gdbreference/commands/set_disassembly-flavor#:~:text=%20VisualGDB%20automatically%20sets%20the%20disassembly%20flavor%20to,the%20GDB%20Startup%20Commands%20list%20for%20your%20project.)

**Comment by zaldwaik** [comment]
I am trying to use the att flavor and used that command in debug console but it has no effect on the "disassembly view". It stays in intel flavor. Any ideas to try? Thanks

**Comment by zaldwaik** [comment]
@jw3215 Thanks. That worked. I guess it has to be done before "disassembly view" is invoked for the first time.

**Comment by a1q123456** [comment]
@zaldwaik 
probably worth adding it to the launch.json under the setupCommands section.
https://code.visualstudio.com/docs/cpp/launch-json-reference#_setupcommands

**Comment by zaldwaik** [comment]
@a1q123456 Thanks. Is there a way to have different launch.json and tasks.jsons files for different programs? They get created under .vscode folder and I have to replace them when I switch to debugging another program.

**Comment by a1q123456** [comment]
Yes, you can create multiple launch targets. You can find the details on the document.

**Comment by zaldwaik** [comment]
Thanks
